app/javascript/pages/ListingDirectory/DirectoryHelpers.tsx

```diff
@@ -190,45 +191,61 @@ export const getTableSubHeader = (listing: RailsRentalListing) => {
   return null
 }
 
+// Get imageCardProps for a given listing
+const getImageCardProps = (listing, hasFiltersSet?: boolean) => ({
+  imageUrl: listing.imageURL,
+  subtitle: `${listing.Building_Street_Address}, ${listing.Building_City} ${listing.Building_State}, ${listing.Building_Zip_Code}`,
+  title: listing.Name,
+  href: `/listings/${listing.listingID}`,
+  tagLabel: listing.Reserved_community_type ?? undefined,
+  statuses: getListingImageCardStatuses(listing, hasFiltersSet),
+})
+
 // Get a set of Listing Cards for an array of listings, which includes both the image and summary table
 export const getListingCards = (listings, directoryType, stackedDataFxn, hasFiltersSet?: boolean) =>
-  listings.map((listing: Listing, index) => (
-    <ListingCard
-      key={index}
-      imageCardProps={{
-        imageUrl: listing.imageURL,
-        subtitle: `${listing.Building_Street_Address}, ${listing.Building_City} ${listing.Building_State}, ${listing.Building_Zip_Code}`,
-        title: listing.Name,
-        href: `/listings/${listing.listingID}`,
-        tagLabel: listing.Reserved_community_type ?? undefined,
-        statuses: getListingImageCardStatuses(listing, hasFiltersSet),
-      }}
-      tableHeaderProps={{
-        tableHeader: getTableHeader(listing),
-        tableSubHeader: getTableSubHeader(listing),
-        stackedTable: true,
-      }}
-      tableProps={{
-        headers: {
-          unitType: { name: "t.units", className: headerClassNames },
-          availability: { name: "t.available", className: headerClassNames },
-          colThree: {
-            name: directoryType === "forRent" ? "t.incomeRange" : "saleDirectory.hoaDues",
-            className: headerClassNames,
-          },
-          colFour: {
-            name: directoryType === "forRent" ? "t.rent" : "saleDirectory.price",
-            className: headerClassNames,
-          },
-        },
-        responsiveCollapse: true,
-        cellClassName: "px-5 py-3",
-        headersHiddenDesktop: ["availability"],
-        stackedData: stackedDataFxn(listing),
-      }}
-      seeDetailsLink={`/listings/${listing.listingID}`}
-    />
-  ))
+  listings.map((listing: Listing, index) => {
+    const hasCustomContent = listing.Reserved_community_type === "Habitat for Humanity"
```

Comment:
not sure what the scale of these kinds of inlined strings are, but I've found placing values like this in a `.env` file or the like helpful

Reply:
yes, good point! I'll at least move it to the top of this file as a const we can re-use elsewhere. We'll also have to put this and others in a map for DAH-1010 which should help

Reply:
just pushed this fix